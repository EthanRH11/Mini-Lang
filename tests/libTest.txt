needs: {
    source: "tests/readFromFile.hmlng"    
    library: "Random"
    library: "Math"
}



    
begin:
    int x = 5;
    int z = 10;

    out_to_console(myFoo(x));
    ...
    out_to_console("The above number should be: 6"); ...

    int y = randomInt(0, 100);
    out_to_console(y);
    ...
    out_to_console(diceRoll(20));
    ...
    out_to_console(generatePin(4));
    ...

    int g = 49;

    out_to_console(sqrt(g));
    ...
end