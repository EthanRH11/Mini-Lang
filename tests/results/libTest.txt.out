6
The above number should be: 6
7
5
8
2
2
9
5